### DISCUSSION TITLE
Will Raspotify work if RPi is connected to iPhone Personal Hotspot

### DISCUSSION THREAD

**Discussion by DINsetting** [dtcooper/raspotify#469] [Q&A] [answered]
I have a unique use case where the RPi won't have an Internet connection. I'd like to be able to have the RPi connect to my iPhone's hotspot and then use the Spotify app on my iPhone to tell the RPi to play a track/playlist.

Is this possible? 

Is there a better way, or something I'm missing?

**Comment by JasonLG1979** [comment, marked as answer]
No. Raspotify requires an internet connection.

**Reply by DINsetting** [reply]
Please bear with me, just to clarify:  Wouldn't the RPi with Raspotify have an internet connection if it was connected to my iPhone's Personal Hotspot via the Pi's integrated WiFi adapter?

**Reply by JasonLG1979** [reply]
It still probably wouldn't work. Most phone hotspots prevent devices from seeing the phone and sometimes even other devices on the hotspot network even though they all still have internet access for security purposes.

**Reply by JasonLG1979** [reply]
For Raspotify to work in zeroconf mode the controlling device has to be able to "see" Raspotify on a local network.

**Reply by JasonLG1979** [reply]
You could using your Spotify credentials and authenticating Raspotify?
